### PR TITLE
Adding support for handling MAPAXES and MAPUNITS

### PR DESCRIPTION
**Pull request by tskille**
adding support in class EGrid for reading MAPAXES and MAPUNITS from egrid file

### CODE REVIEW

File: opm/io/eclipse/EGrid.hpp [hunk 1]
```diff
@@ -67,11 +67,17 @@ class EGrid : public EclFile
 
     const std::vector<std::string>& list_of_lgrs() const { return lgr_names; }
 
+    std::vector<float> get_mapaxes() const { return m_mapaxes; }
```
Comment (akva2) [inline opm/io/eclipse/EGrid.hpp:70]: why the copy?

File: opm/io/eclipse/EGrid.hpp [hunk 2]
```diff
@@ -67,11 +67,17 @@ class EGrid : public EclFile
...
+    std::string get_mapunits() const { return m_mapunits; }
```
Comment (akva2) [inline opm/io/eclipse/EGrid.hpp:71]: why the copy?

File: src/opm/io/eclipse/EGrid.cpp [hunk 3]
```diff
@@ -54,6 +54,7 @@ EGrid::EGrid(const std::string &filename, std::string grid_name) :
     nnc1_array_index = -1;
     nnc2_array_index = -1;
     m_radial = false;
+    m_mapunits = "";
```
Comment (akva2) [inline src/opm/io/eclipse/EGrid.cpp:57]: strings are default empty.

**Comment by tskille** [comment]
This is now ready for review.

**Review by akva2 — CHANGES_REQUESTED** [review]

File: opm/io/eclipse/EGrid.hpp (hunk 2 above)
Comment (tskille) [reply opm/io/eclipse/EGrid.hpp]: fixed const ref on both.

File: src/opm/io/eclipse/EGrid.cpp (hunk 3 above)
Comment (tskille) [reply src/opm/io/eclipse/EGrid.cpp]: deleted line.

**Comment by akva2** [comment]
jenkins build this please

**Review by akva2 — APPROVED** [review]

**Comment by joakim-hove** [comment]
Then we merge :-)

**Pull request merged**